test/validation/api/classification/odp_classification_tests.c

```diff
@@ -214,24 +213,60 @@ int classification_suite_pktv_term(void)
 	return classification_suite_common_term(true);
 }
 
-void configure_cls_pmr_chain(odp_bool_t enable_pktv)
+static void configure_cls_pmr_chain_create_saddr_pmr(int src, const char *saddr)
```

Comment:
I find the 'src' name of the first parameter a bit misleading as it is really the index of the destination CoS. Perhaps you should rename it to 'dst'. Maybe it would be even more clear (?) if the src CoS was a parameter too and the call sites would just pass CLS_DEFAULT in it.

Otherwise this PR looks good.

Reply:
In v3.